Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    bambora-client (0.1.2)
+    bambora-client (0.1.3)
```

Comment:
Strange. Is this always a version behind because the `gem` hasn't been released yet?

Reply:
This *is* strange. I was wondering why we even are maintaining a `Gemfile.lock` in the codebase. The lock file is meant to specify dependencies for a certain machine/environment.

Since this is a gem, dependency versions should be stated in the `*.gemspec` file, and not in the lock file. [This GitHub issue explains the situation well](https://github.com/radar/guides/issues/14), and references an article from Yehuda Katz (albeit from 2010) explaining that the lock file should **not** be checked in for a gem. This is maintained further on [StackOverflow](https://stackoverflow.com/questions/4151495/should-gemfile-lock-be-included-in-gitignore).

A few years ago, Bundler modified the gem setup to include a lock file, [explaining that in some cases, it is desirable to check this in](https://bundler.io/blog/2018/01/17/making-gem-development-a-little-better.html). I don't think our gem falls into this category. I'm spinning up a PR to remove it.